kernel/data-pipeline/scenario/cdc/core/src/main/java/org/apache/shardingsphere/data/pipeline/cdc/core/pojo/CDCJobItemInfo.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.cdc.core.pojo;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.shardingsphere.data.pipeline.core.job.JobStatus;
+import org.apache.shardingsphere.data.pipeline.core.job.progress.JobItemIncrementalTasksProgress;
+
+/**
+ * CDC job item info.
+ */
+@RequiredArgsConstructor
+@Getter
+public class CDCJobItemInfo {
+    
+    private final int shardingItem;
+    
+    private final String dataSourceName;
+    
+    private final String tableNames;
+    
+    private final JobStatus status;
+    
+    private final boolean active;
+    
+    private final long inventoryRecordsCount;
+    
+    private final long processedRecordsCount;
+    
+    private final int inventoryFinishedPercentage;
+    
+    private final JobItemIncrementalTasksProgress incremental;
+    
+    private final String confirmedPosition;
+    
+    private final String currentPosition;
+    
+    private final String errorMessage;
+}
```

Comment:
Could we embed `TransmissionJobItemInfo` and just keep 2 new fields (`confirmedPosition` and `currentPosition`)?